### PR TITLE
SDL2 image: Add features for libjpeg, tiff and webp

### PR DESCRIPTION
**Pull request by Ghabry**
(kept PNG always on, most useful format I guess)

### CODE REVIEW

File: ports/sdl2-image/CMakeLists.txt [hunk 1]
```diff
@@ -7,11 +7,14 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
...
 
+# Forwarding the argument from the portfile replaces the ; in the lists with whitespace, convert it back
+string (REPLACE " " ";" SDL_IMAGE_DEPENDENCIES "${SDL_IMAGE_DEPENDENCIES}")
+
 # enable all file formats which are supported through external dependencies
 # first try to load them statically (lib file in vcpkg installation)
 # if this fails try to make them a dynamic dependency (dll will be loaded at runtime) if possible. vcpkg cannot resolve these dependencies!
 # else do not support this file format at all
-set(DEPENDENCIES PNG JPEG TIFF WEBP)
+set(DEPENDENCIES ${SDL_IMAGE_DEPENDENCIES})
```
Comment (carstene1ns) [inline ports/sdl2-image/CMakeLists.txt]: Maybe the above hack is needed, because this is not a list? Just thinking.

Comment (Ghabry) [reply ports/sdl2-image/CMakeLists.txt]: That's a good point, will check this.

Comment (Ghabry) [reply ports/sdl2-image/CMakeLists.txt]: checked this and the way set is used in the old code creates a list, too.

**Comment by ras0219-msft** [comment]
Thanks for the PR and sorry for the incredibly long delay!

**Pull request merged**